dd-sdk-android-ndk/src/androidTest/kotlin/com/datadog/android/ndk/NdkTests.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.ndk
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.gson.JsonParser
+import fr.xgouchet.elmyr.junit4.ForgeRule
+import java.lang.RuntimeException
+import java.nio.charset.Charset
+import java.util.regex.Pattern
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+internal class NdkTests {
+
+    @get:Rule
+    val forge = ForgeRule()
+
+    @get:Rule
+    val temporaryFolder = TemporaryFolder()
+
+    companion object {
+        init {
+            System.loadLibrary("datadog-native-lib")
+            System.loadLibrary("datadog-native-lib-test")
+        }
+    }
+
+    @Test
+    fun ndkUnitTests() {
+        if (runNdkUnitTests() != 0) {
+            throw RuntimeException("NDK Unit tests failed")
+        }
+    }
+
+    @Test
+    fun signalHandlerIntegrationTest() {
+
+        // clear the logs dir
```

Comment:
Is it a TODO ? Or a remain of previous code ?

Reply:
Remain